static/src/server.ts

```diff
@@ -105,6 +104,8 @@ const BARD_ALLOWED_CHARTS = new Set(["LINE", "BAR", "RANKING", "SCATTER"]);
 // The root to use to form the dc link in the tile results
 // TODO: update this to use bard.datacommons.org
 const DC_URL_ROOT = "https://datacommons.org/explore#q=";
+// Size of the PNG to return for the chart query
+const PNG_WIDTH = 1600;
```

Comment:
Would it make sense to return SVGs at the same width (maybe set this in the SVG chart itself?) Now SVGs get returned with a 550px width and pngs with 1600

Reply:
The reason for returning pngs at 1600 is to just use a really large size to preserve image quality no matter what size Bard actually chooses to render the image at. I tried drawing the actual svg at 1600 and the scaling makes everything super hard to see. Instead, I updated the returned svg to use viewbox with the original dimensions & caller can then update the width and height to whatever they need.

Reply:
Sounds reasonable